# Flux test

In [2]:
import numpy as np
import matplotlib.pyplot as plt
from scipy.stats import lognorm
import matplotlib.style
import matplotlib as mpl
import math

#Set default figure size
mpl.rcParams['figure.figsize'] = [12.0, 8.0]
mpl.rcParams["legend.frameon"] = False
mpl.rcParams.update({'font.size': 22})

#Could problem also switch these to rcParams calls
plt.rc('text', usetex=True)
#plt.rc('font', family='serif')

From arXiv:2109.11049v2 the flux of neutrinos from the spallation neutron source is
$$F_{SNS}=4.7 \times10^7 \nu cm^{−2} s^{−1}$$
at a distance of 20m.

1 cm$^2$ at a distance of 20m is a solid angle, $\Omega = \frac{A}{r^2} = \frac{1}{2000^2} Sr$

So the flux of neutrinos from the SNS is
$$F_{SNS} = 19 \times10^{14} \nu Sr^{-1} s^{-1}$$


From Leo's PPFX Paper arXiv:1607.00704v3 the neutrino flux is
$$F_{NuMI} = 287 \pm 22 \nu_\mu /m^2 / 10^6 POT$$

In [4]:
potPerWeek=20e18
secondsPerWeek=60*60*24*7

potPerWeek/secondsPerWeek

Fnumi=287*(potPerWeek/1e6)/secondsPerWeek
print(Fnumi/1e9)

9.49074074074074


$F_{NuMI}=9.5 \times 10^9 \nu_\mu /m^2 / s$

1 m$^2$ at a distance of 1000m is a solid angle, $\Omega = \frac{A}{r^2} = \frac{1}{1000^2} Sr$

$F_{NuMI}=9.5 \times 10^{15} \nu_\mu / Sr / s$